spec/miq_ae_object_spec.rb

```diff
@@ -316,4 +316,23 @@ def value_match(value, xml_value)
       end.to raise_exception(MiqPassword::MiqPasswordError)
     end
   end
+
+  context "integer" do
+    let(:value) { "45" }
+    let(:datatype) { "Integer" }
+    let(:datatype1) { "integer" }
+    let(:datatype2) { "Fixnum" }
+
+    it "returns value to_i" do
+      expect(described_class.convert_value_based_on_datatype(value, datatype)).to eq(45)
```

Comment:
Could we just loop over the different datatypes in one test?  There's not a big difference between the first test `returns value to_i` and the other two `returns value to_i` so maybe we just combine them.